test/test_amity.py

```diff
@@ -0,0 +1,415 @@
+        self.of.add_person(self.fel)
+        self.assertEqual(self.fel, self.of.occupants[0])
+
+    def test_can_add_fellow_to_livingroom_room(self):
+        """
+        Should be able to add fellow to LivingRoom rooms
+        """
+        self.lroom = livingroom.LivingRoom("Kitchen")
+        self.fellow = fellow.Fellow("May", "Teresa")
+        self.lroom.add_person(self.fellow)
+        self.assertEqual(self.fellow, self.lroom.occupants[0])
+
+    def test_can_add_staff_to_office_room(self):
+        """
+        Should be able to add staff to Office room
+        """
+        self.staff = staff.Staff("TT", "PP")
+        self.office = office.Office("Main Office")
+        self.office.add_person(self.staff)
+        self.assertEqual(self.staff, self.office.occupants[0])
+
+    def test_livingroom_raise_error_for_addition_of_staff(self):
+        """
+        LivingRoom should raise an InvalidRoomOccupantError exception when Staff tries to join it.
+        """
+        self.staff = staff.Staff("TT", "PP")
+        self.lroom = livingroom.LivingRoom("Livingroom")
+        with self.assertRaises(InvalidRoomOccupantError):
+            self.lroom.add_person(self.staff)
+
+    def test_can_remove_fellow_from_office(self):
+        """
+        Should be able to remove a fellow from office room
+        """
+        self.office = office.Office("Hogwarts")
+        self.f1 = fellow.Fellow("Nan", "Pi")
+        self.f2 = fellow.Fellow("KK", "Brown")
+        self.office.add_person(self.f1)
+        self.office.add_person(self.f2)
+
+        self.assertTupleEqual(
+            (self.office.remove_person(self.f2), self.office.remove_person(self.f1)),
+            (self.f2, self.f1)
+        )
+
+    def test_can_remove_fellows_staff_from_office(self):
+        """
+        Should be able to remove Fellows and Staff from office
+        """
+        self.f1 = fellow.Fellow("Mark", "Mike")
+        self.f2 = fellow.Fellow("JJ", "PP")
+        self.s1 = staff.Staff("FF", "KK")
+        self.s2 = staff.Staff("G", "FF")
+        self.office = office.Office("Small Office")
+        self.office.add_person(self.f1)
+        self.office.add_person(self.f2)
+        self.office.add_person(self.s1)
+        self.office.add_person(self.s2)
+
+        self.assertTupleEqual(
+            (self.office.remove_person(self.s1), self.office.remove_person(self.s2), self.office.remove_person(self.f2),
+             self.office.remove_person(self.f1),),
+            (self.s1, self.s2, self.f2, self.f1)
+        )
+
+    def test_raises_exception_when_removing_non_person_type(self):
+        """
+        Should raise TypeError exception when a non-Person type is removed from the room
+        """
+        self.office = office.Office("Strange Office")
+        self.fellow = fellow.Fellow("Cool", "Guy")
+        with self.assertRaises(TypeError):
+            self.office.remove_person('Cool Guy')
+
+
+class TestAmitySystem(unittest.TestCase):
```

Comment:
Modularize your tests i.e have a different test file for each class. create modules i.e have Room module that has its functionality and its own test file. You can check out my repo for a sample(andela-landia)